openwisp_firmware_upgrader/api/tests/base/__init__.py

```diff
@@ -0,0 +1,41 @@
+from django.contrib.auth import get_user_model
+from django.test import Client
+from django.urls import reverse
+from openwisp_firmware_upgrader.tests.base import TestUpgraderMixin
+from swapper import load_model
+
+from openwisp_users.tests.utils import TestMultitenantAdminMixin
+
+BatchUpgradeOperation = load_model('firmware_upgrader', 'BatchUpgradeOperation')
+Build = load_model('firmware_upgrader', 'Build')
+Category = load_model('firmware_upgrader', 'Category')
+DeviceFirmware = load_model('firmware_upgrader', 'DeviceFirmware')
+FirmwareImage = load_model('firmware_upgrader', 'FirmwareImage')
+UpgradeOperation = load_model('firmware_upgrader', 'UpgradeOperation')
```

Comment:
do not load the concrete models here, this may cause issues because this file will be imported also by extension apps (eg: the sample app)

Reply:
I've moved them back to test_views.py, let me know if this is the correct place